google-map-marker.html

```diff
@@ -366,7 +366,7 @@
         subtree: true
       });
 
-      var content = this.innerHTML.trim();
+      var content = this.children[0];
```

Comment:
(I'm not the author of this pull request, just interested in getting the issue fixed)

I've tried this line (369) with all four combinations of shady dom, shadow dom, webcomponents and webcomponents-lite and it certainly seems to work. Why might it fail under shadow dom?

On the other hand, issues:

The next line, `370 if (content) { ... }`, fails because `content` always contains the new div element now so `content` itself is always true. This leads to opening an empty infowindow.  Maybe `370 if (content.innerHTML.trim()) { ... }`

_More serious_, Polymer ink bearing elements crash the infowindow.  For example, when a paper-button or paper-icon-button is clicked inside the infowindow, elements are added inside the button to show the ink effect. This triggers `_contentChanged()` in the middle of the ink actions, and the infowindow content winds up empty.  Non ink elements don’t seem to have this problem, for example an iron-icon with an on-tap event works fine.